### DISCUSSION TITLE
libcjose RPM for 2.4.x series?

### DISCUSSION THREAD

**Discussion by jlbooker** [OpenIDC/mod_auth_openidc#1149] [Q&A] [answered]
Hello,

I'm installing mod_auth_openidc 2.4.14.4-1 on a RHEL 7 host (from `mod_auth_openidc-2.4.14.4-1.el7.x86_64.rpm`), but I'm missing the `cjose` and `libcjose.so` dependencies. I've looked through the recent releases in the 2.4 series, but I'm not seeing a libcjose RPM package. I found the .deb package on the 2.4.0 release [libcjose0_0.6.1.5-1](https://github.com/OpenIDC/mod_auth_openidc/releases/download/v2.4.0/libcjose0_0.6.1.5-1.bionic+1_amd64.deb), but there's no corresponding RPMs. Am I missing something, or could someone point me to the EL7 RPMs?

**Comment by zandbelt** [comment, marked as answer]
https://github.com/OpenIDC/cjose/releases

**Reply by jlbooker** [reply]
Ah! In their own separate project. Perfect. Thanks so much!
